palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -1,10 +1,12 @@
+import os, \
+    sys, \
+    yaml
```

Comment:
yaml is unused in this file, remove the import

Reply:
removed